lib/releaselib.sh

```diff
@@ -833,3 +846,51 @@ release::docker::release () {
     logrun $GCLOUD config set account $USER@google.com
   fi
 }
+
+###############################################################################
+# Determine whether the release was built with Bazel.
+# @param kube_root - Root of kubernetes tree
+# @return 0 if built with Bazel, 1 otherwise
+release::was_built_with_bazel() {
```

Comment:
Got it.  Then I'll be pedantic and ask that you update the function description or otherwise add a comment to that effect.   It's now obvious what you are doing with the code, but it wasn't at first glance.